astroquery/esa_hubble/tests/dummy_handler.py

```diff
@@ -0,0 +1,82 @@
+import os
+from six.moves import urllib
+from astroquery.utils.tap.model import modelutils
+
+__all__ = ['ESAHubble', 'ESAHubbleClass', 'Conf', 'conf',
+           'DummyHandler', 'dummy']
+
+
+def data_path(filename):
+    data_dir = os.path.join(os.path.dirname(__file__), 'data')
+    return os.path.join(data_dir, filename)
+
+
+class DummyHandler(object):
+
+    def get_file(self, url, filename, verbose=False):
+        print("************* DummyHandler!!!!!!!!")
+        file = data_path(filename)
+        print(file)
+        if file.endswith(".xml"):
+            with open(file, 'r') as myfile:
+                data = myfile.read().replace("\n", "")
+        else:
+            with open(file, 'rb') as myfile:
+                data = myfile.read()
+        return data
+
+    def get_table(self, url, filename=None, output_format='votable',
+                  verbose=False):
+        print("************* DummyHandler!!!!!!!!")
+        if filename is None:
+            raise ValueError("filename must be specified")
+        urllib.request.urlretrieve(url, filename)
```

Comment:
I'm unclear why this line is here.  The "DummyHandler" seems to exist for testing purposes, yet this line calls a `urlretrieve`, which requires internet access and therefore is attempting to download the real file.  Could you clarify the purpose of this handler?  If it is accessing web-based resources, it cannot be used in non-remote tests.